associations/views/association.py

```diff
@@ -65,3 +65,13 @@ def get_serializer_class(self):
             return AssociationShortSerializer
         else:
             return AssociationSerializer
+
+def upload_association_logo(request, association_pk):
+    if request.method == 'POST' and request.FILES['myfile']:
+        myfile = request.FILES['myfile']
+        fs = FileSystemStorage()
+        filename = fs.save(myfile.name, myfile)
+        uploaded_file_url = fs.url(filename)
+        return render(request, 'core/simple_upload.html', {
+            'uploaded_file_url': uploaded_file_url
+        })
```

Comment:
Er, it's weird to see `upload_association_logo` here, but I guess it's a remnant from the other branch

Reply:
Hum true 😅